src/coreclr/tools/aot/crossgen2/Program.cs

```diff
@@ -523,6 +523,9 @@ private int Run(string[] args)
                             _commandLineOptions.CompileBubbleGenerics);
                     }
 
+                    // Load any profiles generated by method call chain analyis
+                    CallChainProfile jsonProfile = new CallChainProfile(_commandLineOptions.CallChainProfileFile, _typeSystemContext, referenceableModules);
```

Comment:
This crashes when CallChainProfileFile is not specified, could you please add a string.IsNullOrEmpty conditional check? I guess your current PR build will also fail because of that.

Reply:
Thanks for saving a couple hours of CI time 😉